pkg/sriov/sriov.go

```diff
@@ -169,16 +169,21 @@ func (s *sriovManager) ReleaseVF(conf *sriovtypes.NetConf, podifName string, cid
 			return fmt.Errorf("failed to rename link %s to host name %s: %q", podifName, conf.OrigVfState.HostIFName, err)
 		}
 
+		linkObj, err = s.nLink.LinkByName(conf.OrigVfState.HostIFName)
+		if err != nil {
+			return fmt.Errorf("failed to get netlink original device with name %s: %q", podifName, err)
+		}
+
 		// reset effective MAC address
 		if conf.MAC != "" {
 			hwaddr, err := net.ParseMAC(conf.OrigVfState.EffectiveMAC)
 			if err != nil {
 				return fmt.Errorf("failed to parse original effective MAC address %s: %v", conf.OrigVfState.EffectiveMAC, err)
 			}
 
-			if err = s.nLink.LinkSetHardwareAddr(linkObj, hwaddr); err != nil {
-				return fmt.Errorf("failed to restore original effective netlink MAC address %s: %v", hwaddr, err)
-			}
+			err = utils.Retry(5, 200*time.Millisecond, func() error {
```

Comment:
Should the CNI at least output a warning message if this fails?

Reply:
today there is no way to output a warning in the CNI only via errors as this is called by multus that is called by crio

Reply:
we return error sorry I miss that :)